R/bpvalidate.R

```diff
@@ -132,7 +132,12 @@ setMethod("show", "BPValidate", function(object) {
     ## (only if the function/expr depends on the global)
     inpath <- list()
     if (length(unknown) && identical(.GlobalEnv, env)) {
-        inpath <- lapply(unknown, function(x) head(find(x), 1))
+        inpath <- lapply(unknown, function(x) {
+            finds <- find(x)
+            ## exclude internal objects
+            finds <- finds[(!startsWith(finds, "."))|finds == ".GlobalEnv"]
```

Comment:
Was wondering whether a better strategy is to 'clear' -- 
```
> search()
 [1] ".GlobalEnv"        "package:devtools"  "package:usethis"
 [4] "package:stats"     "package:graphics"  "package:grDevices"
 [7] "package:utils"     "package:datasets"  "package:methods"
[10] "Autoloads"         "package:base"
> devtools::load_all()
ℹ Loading hcaCaseStudies
> search()
 [1] ".GlobalEnv"             "devtools_shims"         "package:hcaCaseStudies"
 [4] "package:testthat"       "package:devtools"       "package:usethis"
 [7] "package:stats"          "package:graphics"       "package:grDevices"
[10] "package:utils"          "package:datasets"       "package:methods"
[13] "Autoloads"              "package:base"
```
anything that starts with "package:" instead of 'block' things that start with `.`?